ingest/build-configs/ncbi/bin/transform-segment

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+Transforms numeric segments to corresponding segment names of
+Influenza A viruses in the NDJSON record's `segment` field.
+"""
+import argparse
+import json
+from sys import stderr, stdin, stdout
+
+
+def parse_args():
```

Comment:
```suggestion
def parse_args() -> argparse.Namespace:
```

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/40/commits/eb5e068ef2d4260b01c7ac49fa805b539fb05e83